SECP-service/src/main/java/com/visucius/secp/DTO/FilterCreateRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.visucius.secp.DTO;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Set;
+
+public class FilterCreateRequest {
+
+    @JsonProperty
+    public String name;
```

Comment:
Just for consistencies sake (since most of the other DTO classes are like this), I think we should put a line break between each property here.

Reply:
Good call